.github/workflows/native-image.yaml

```diff
@@ -35,10 +36,12 @@
           key: ${{ runner.os }}-graalvm-${{env.GRAALVM_VERSION}}
           restore-keys: |
             ${{ runner.os }}-graalvm-
-      - uses: DeLaGuardo/setup-graalvm@8bbfe44ef9c6f5c07e5af036a1bffd561c037d18
+      - uses: DeLaGuardo/setup-graalvm@5.0
         with:
-          graalvm-version: ${{env.GRAALVM_VERSION}}
-      - run: ./mvnw package -Dnative -DskipTests $([ $(uname -s) = Linux ] && echo "-Dgraalvm.static=--static") -Dcbi.jarsigner.skip=true
+          graalvm: ${{env.GRAALVM_VERSION}}
+          java: ${{env.GRAALVM_JAVA}}
+      - run: ./mvnw -B package -Dnative -DskipTests $([ $(uname -s) = Linux ] && echo "-Dgraalvm.static=--static") -Dcbi.jarsigner.skip=true
```

Comment:
Could you add modify the term : 
`"-Dgraalvm.static=--static"`
to be
`"-Dgraalvm.static=--static -H:+StaticExecutableWithDynamicLibC"`.

Just need that extra flag on the end.

Reply:
The runner reports it as an unidentified option. Did I type it wrong?

Reply:
It may need to be something like `'-Dgraalvm.static="--static -H:+StaticExecutableWithDynamicLibC"'` but I'm not 100% sure. It's a quoting issue.

Reply:
How about just `"-Dgraalvm.static=-H:+StaticExecutableWithDynamicLibC"` .

Reply:
That works